src/oc/web/stores/user.cljs

```diff
@@ -67,3 +71,53 @@
 (defmethod dispatcher/action :login-with-email/success
   [db [_]]
   (dissoc db show-login-overlay-key))
+
+;; Auth actions
+
+(defmethod dispatcher/action :bot-auth
```

Comment:
This function is an action. It doesn't change the app state and uses functions unrelated to the app state, `router/get-token`, etc.  We should move this to user actions.  The current user action dispatches but doesn't need to and can just call this code.

Reply:
Done